javascript/ql/test/tutorials/cookbook/EvalTaint/EvalTaint.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Taint-tracking to 'eval' calls
+ * @description Tracks user-controlled values into 'eval' calls.
```

Comment:
Perhaps add a reference to `js/code-injection` to prevent readers from adapting this query to reinvent the wheel.